### ISSUE TITLE
Is there a live demo?

### ISSUE THREAD

**Issue by tvvocold** [SemihY/musicnews#4]
:)

**Comment by SemihY** [comment]
Yes :) but it is uncompleted(Login panel etc . )In my spare time i am thinking to do it.

**Comment by tvvocold** [comment]
check this : https://github.com/upvote/upvote

**Comment by SemihY** [comment]
But it is coded with ruby. I coded this project for working with javascript especially meteor framework.therefore i also check this
